public/templates/cookies_tab.html

```diff
@@ -0,0 +1,12 @@
+<div class="cookies tab">
+    <h1>Cookies</h1>
+    <p class="description">
+        This is the cookies are saved on your system and it contains the preferences
+        you chose in the settings page
+    </p>
+    <input type="text" name="cookie_field" value="" readonly />
+    <p class="description">
+        The cookies stored are not used by us for any malicious intend or for
+        tracking you in any way.
```

Comment:
```suggestion
        The cookies stored are not used by us for any malicious intent or for tracking you in any way.
```

Reply:
Thank you for making this change is that this will make the code a little less readable because the sentence will be very long. Anyone working on this file will have to scroll horizontally to see the whole sentence, which is not good, What do you say?